media-fonts/fontawesome/fontawesome-5.9.0.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_PN="Font-Awesome"
+inherit font
+
+DESCRIPTION="The iconic font"
+HOMEPAGE="https://fontawesome.com"
+SRC_URI="https://github.com/FortAwesome/${MY_PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="CC-BY-3.0 OFL-1.1"
+SLOT="0/5"
+KEYWORDS="amd64 ~arm arm64 x86"
```

Comment:
you have to drop all keywords to `~` on bump.

Reply:
I mean `amd64` should be `~amd64` and so on.

Reply:
Done :)